build/CodeCoverage.runsettings

```diff
@@ -1,40 +1,44 @@
-<?xml version="1.0" encoding="utf-8"?>  
+<?xml version="1.0" encoding="utf-8"?>
 
 <!-- Customised run settings file to exclude test assemblies from coverage.
     See https://msdn.microsoft.com/en-us/library/jj159530.aspx for more info. -->
 
-<!-- File name extension must be .runsettings -->  
-<RunSettings>  
-  <DataCollectionRunSettings>  
-    <DataCollectors>  
-      <DataCollector friendlyName="Code Coverage" uri="datacollector://Microsoft/CodeCoverage/2.0" assemblyQualifiedName="Microsoft.VisualStudio.Coverage.DynamicCoverageDataCollector, Microsoft.VisualStudio.TraceCollector, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">  
-        <Configuration>  
-          <CodeCoverage>  
-  
-<!--  
+<!-- File name extension must be .runsettings -->
+<RunSettings>
+    <DataCollectionRunSettings>
+        <DataCollectors>
+            <DataCollector friendlyName="Code Coverage" uri="datacollector://Microsoft/CodeCoverage/2.0" assemblyQualifiedName="Microsoft.VisualStudio.Coverage.DynamicCoverageDataCollector, Microsoft.VisualStudio.TraceCollector, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+                <Configuration>
+                    <CodeCoverage>
+
+                        <!--  
 About include/exclude lists:  
 Empty "Include" clauses imply all; empty "Exclude" clauses imply none.  
 Each element in the list is a regular expression (ECMAScript syntax). See http://msdn.microsoft.com/library/2k3te2cs.aspx.  
 An item must first match at least one entry in the include list to be included.  
 Included items must then not match any entries in the exclude list to remain included.  
--->  
-  
-            <!-- Match assembly file paths: -->  
-            <ModulePaths>  
-              <Include>  
-                <ModulePath>.*\microsoft.identitymodel.clients.activedirectory.dll$</ModulePath>                  
-                <ModulePath>.*\microsoft.identity.core.dll$</ModulePath>
-                <ModulePath>.*\microsoft.identity.client.dll$</ModulePath>
-
-              </Include>  
-              <Exclude>  
-              </Exclude>  
-            </ModulePaths>  
-    
-          </CodeCoverage>  
-        </Configuration>  
-      </DataCollector>  
-    </DataCollectors>  
-  </DataCollectionRunSettings>  
-</RunSettings>  
-  
\ No newline at end of file
+-->
+                        <ModulePaths>
+                            <Include>
+                                <ModulePath>.*\microsoft.identity.client.dll$</ModulePath>
+                            </Include>
+                            <Exclude>
```

Comment:
There are still quite a lot of test projects included - 
![image](https://github.com/user-attachments/assets/1519d4f4-f627-4128-9ee9-b9e73c02bd96)

Reply:
Discussed offline, Exclusions seem not to function in our build due to complexities. Additional drives are scanned and report incorrect results.

Reply:
After additional testing, this issue is now resolved